cavern/src/main/java/org/opencadc/cavern/files/GetAction.java

```diff
@@ -137,13 +143,14 @@ public void doAction() throws Exception {
             Files.copy(source, out);
             log.debug("Completed copy of file " + source);
             out.flush();
-        } catch (FileNotFoundException | NoSuchFileException e) {
-            log.debug(e);
-            syncOutput.setCode(404);
-        } catch (AccessControlException | AccessDeniedException e) {
-            log.debug(e);
-            syncOutput.setCode(403);
+        }
+        catch (NodeNotFoundException e) {
+            log.debug("404 error with GET: ",  e);
+            sendError(404, e.getLocalizedMessage());
```

Comment:
Same here, can just throw ResourceNotFoundException.

Curious about the removal removal of FileNotFound & NoSuchFileException though...

Reply:
added those other exceptions back in as at this point I don't know if they are run time exceptions or not. Added a check for container nodes because Files.copy was throwing and IOException for action on a container, which was coming out as a 500.